src/js/cookie-policy.js

```diff
@@ -22,25 +22,27 @@ cpNs.cookiePolicy = (function() {
       let content = options.content;
       let duration = options.duration;
       let start = `
```

Comment:
IMHO this is a little weird to do it this way. I'd include the `content` variable in this template and merge this with the `end` template.

Reply:
Not sure what you mean. The content variable is null checked and set to a default value below this. This could be pulled up but its just changing the order of the concatenation.